website/frontend/templates/docs/build-windows.html

```diff
@@ -6,113 +6,192 @@
-        </li>
-        <li>Download "libdiscid-0.6.1-win32.zip" and put "discid.dll" in your Windows\System32 folder</li>
+        <li>PyQt 5.7.1 or later</li>
+        <li>Mutagen 1.37 or later</li>
+        <li>discid</li>
       </ul>
 
-      <h3 id="pip">Pip for Windows</h3>
+      <p>
+         The easiest way to install them is to open an Administrator command prompt, navigate to the Picard source directory and type:
+      </p>
+      <pre>pip3 install -r requirements.txt</pre>
+
+      <p>
+         Alternatively to install them individually run:
+      </p>
+      <pre>pip3 install PyQt>=5.7.1
+pip3 install Mutagen>=1.37
+pip3 install discid</pre>
+
+      <h2 id="python_picard1">Python for Picard version 1.4.x</h2>
+
+      <h3 id="python27">Python 2.7.x</h3>
 
       <ul>
-        <li> <a rel="nofollow" href="https://sites.google.com/site/pydatalog/python/pip-for-windows">https://sites.google.com/site/pydatalog/python/pip-for-windows</a>
-        </li>
-        <li>Download to a location you find again (no installation)
+        <li>
+          Download and install Python 2.7.x for Windows from 
+          <a rel="nofollow" target="_blank" href="http://python.org/downloads/">http://python.org/downloads/</a>
         </li>
-        <li>in Pip_Win:
-          <ul>
-            <li>
-              <code>pip install mutagen</code>for Mutagen (1.22 tested)
-            </li>
-            <li>
-              <code>pip install discid</code>for python-discid (1.1.x tested)
-            </li>
-          </ul>
+        <li>
+          Python should add itself to your %PATH% environment variable, 
+          but if not add both the Python install directory and its scripts subdirectory to %PATH%
         </li>
       </ul>
 
-      <h3 id="gettext">gettext</h3>
+      <h3 id="python2_packages">Python 2 packages</h3>
+      
+      <p>
+         You need need the following additional python packages to run Picard 1.4.x from source:
+      </p>
 
       <ul>
-        <li> <a rel="nofollow" href="http://gnuwin32.sourceforge.net/packages/gettext.htm">http://gnuwin32.sourceforge.net/packages/gettext.htm</a>
-        </li>
-        <li>Add the gettext folder to&#160;%PATH% (normally <i>C:\Programs\GnuWin32\bin</i>)
+        <li>PyQt 4.10 or later</li>
+        <li>Mutagen 1.37 or later</li>
+        <li>discid</li>
+      </ul>
+      
+      <p>
+        The easiest way to install them is using PIP. 
+        PIP is included as standard in Python 2.7.9 or later - so this is a good reason to upgrade if you
+        already have an earlier version of Python 2.7 installed.
+        Alternatively you can install PIP separately on earlier versions or install the above packages manually.
+      </p>
+        
+      <p>
+        Assuming that you want to use PIP, open an Administrator command prompt, navigate to the Picard source directory and type:
+      </p>
+      
+      <pre>pip install PyQt4>=4.10
```

Comment:
I have not tested this (because I don't have a clean PC to test it on), but I did research it. The latest PyPi is 4.11.4 and the Riverbank Download Page for 4.11.4 is 32- or 64-bit installer which AFAIK include binaries, whereas the 4.12 version referred to as not having binaries is only downloadable as a zip file. So I am reasonably certain that it includes binaries.